src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { useAppSelector } from '../../hooks/useAppSelector';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { useAppDispatch } from '../../hooks/useAppDispatch';
+import { currentTodoSlice } from '../../features/currentTodo';
```

Comment:
you can export all actions (using named export) separately from the slice file and import them here